Interfaces/__init__.py

```diff
@@ -0,0 +1,19 @@
+#  Drakkar-Software OctoBot-Tentacles
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+
+from .interfaces import *
+from .services import *
```

Comment:
Why the root folder is named `Interfaces` ?

Reply:
I imagined something like 3 folder at root next to `Trading, Websockets`... like `Services, Interfaces, Notifications`.

Reply:
Because that's the best name I found to describe what is going on in this tentacle folder, if you have any suggestion, no problem 😄

Reply:
I don't understand why there is no :
- Interfaces/ --> where interface tentacles are located
- Services/ --> where service tentacles are located
- Notifications/ --> where notification tentacles are located

Reply:
No reason, you are right, better like this